Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "openai_api_rust"
-version = "0.1.8"
+version = "0.1.9"
 authors = ["i@guantong.dev"]
```

Comment:
You could update this to add you as the author.
And please help me update i@guantong.dev to i@gt.email, thanks!

Reply:
Done!